website/docs/docs/dbt-versions/release-notes/04-Sept-2023/ci-updates-phase3-rn.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Update: Improvements to dbt Cloud continuous integration"
+description: "September 2023: Improved deletion of temporary schemas"
+sidebar_label: "Update: Improved deletion of temporary schemas"
+tags: [Sept-2023, CI]
+date: 2023-09-18
+sidebar_position: 08
+---
+
+Temporary schemas are now being deleted (dropped) for all adapters (like Databricks), PrivateLink connections, and environment variables in connection strings. 
+
+dbt Labs has rearchitected how schema deletion works for [continuous integration (CI)](/docs/deploy/continuous-integration) runs. We created a new service to delete any schema with a prefix of `dbt_cloud_pr` that's been generated by a PR run.
+
+However, temporary schemas will not be automatically deleted if:
+- Your project overrides the `generate_schema_name` macro but it doesn't contain the prefix `dbt_cloud_pr`. 
```

Comment:
```suggestion
- Your project overrides the `generate_schema_name` macro but it doesn't contain the prefix `dbt_cloud_pr_`. 
```

One small suggestion here: the prefix for the schema needs to end with an underscore.